packages/mc-html-template/README.md

```diff
@@ -36,10 +36,23 @@ At the moment we define the following placeholders:
 - `__DATALAYER_JS__`: the initial configuration for GTM, in case the `trackingGtm` is defined in the `env.json`
 - `__GTM_SCRIPT__`: the actual GTM script, in case the `trackingGtm` is defined in the `env.json`
 
-#### `loadEnv(configPath: String): Object`
+#### `loadEnv(configPath: String, { disableCache: boolean } = { disableCache: false }): Object`
```

Comment:
I would leave the optional param out, since it's just for testing purposes.

Reply:
Okay sure.

Reply:
6396711